app/views/layouts/_revised_messages.html.haml

```diff
@@ -1,20 +1,21 @@
-/ Note: 
-/   We primarily use 3 flash types - success, error, and notice
-/   success - No actionable info. Basically, Bike Index did what you asked. Auto faded
-/   error   - Messages because you did something wrong. Not faded
-/   notice  - Messages that don't fit in the above categories. Not faded, may have actionable info
+-# Note:
+-# -----
+-# We primarily use 3 flash types - success, error, and notice
+-# success - No actionable info. Basically, Bike Index did what you asked. Auto faded
+-# error   - Messages because you did something wrong. Not faded
+-# notice  - Messages that don't fit in the above categories. Not faded, may have actionable info
```

Comment:
Why switch comment types?

Reply:
In Haml the slash renders as an HTML comment so it ends up rendered for the client

Reply:
Oooh. Because it won't be sent to the client. Nifty! Good addition 👍